src/api/posts.js

```diff
@@ -11,7 +11,8 @@ export default async (subreddit) => {
     const URL = `${config.BASE_URL}search/submission/${query}`;
     const res = await axios.get(URL);
 
-    if (res.data.data.length > 0) {
+    if (res.data.length > 0) return res.data;
```

Comment:
You added this just to make the tests work, right? That's an antipattern. You should never (never say never :-D) change your implementation so that it fits a test. Except there's a but of course

Reply:
Yeah, I added this just for tests, when writing tests I shouldn't change things just for tests to work right?

Reply:
Exactly :+1: